### ISSUE TITLE
[MediaBundle] Rename getThumbnails to getFormats

### ISSUE THREAD

**Issue by sulu-io-user** [sulu/sulu#446]
<a href="https://github.com/dantleech"><img height="50" width="50" src="https://avatars.githubusercontent.com/u/530801?v=2" /></a>
By @dantleech _from_ https://github.com/sulu-cmf/SuluMediaBundle/issues/75

The `getThumbnails` method `Api\Media` returns `$formats`. It is more appropriate and less confusing to call this method `getFormats`.

Thumbnail would be inappropriate if the image were transformed in such a way that the result wouldn't resemble a thumbnail.

**Comment by danrot** [comment]
https://github.com/sulu-cmf/sulu/blob/develop/src/Sulu/Bundle/MediaBundle/Api/Media.php#L444 should do it, shouldn't it?

**Comment by dantleech** [comment]
Well, `getThumbnails` still exists. It should be removed.

**Comment by kfrohwein** [comment]
Hi
looking at this I can say it will be difficult to change that because to me it seems this will break the theming api.
```php
<img style="width: 50px;" src="{{ media.thumbnails['50x50'] }}"/>
```

The change itself is very simple but I don't know how you handle API Changes. And the comment to wait till the annotation works seems a little troublesome to me. I mean the annotation just checks if it is a string. So why wait?

**Comment by danrot** [comment]
Yeah, you are definitely right about the BC argument. We won't be able to change this before 2.0, so I will tag it with that milestone. And what annotation are you talking about?

**Comment by chirimoya** [comment]
closed in favor of #802
